PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,38 @@
-### Checklist
+<!--
+  You are amazing! Thanks for contributing to our Step library!
+  Please fill this template with the details of your change.
+-->
 
-- [ ] I've read and accepted the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
-- Requires _MAJOR / MINOR / PATCH_ [version update](https://semver.org/)
+### Checklist
+<!--
+  Put an `x` in the boxes that apply. You can also fill these out after
+  creating the PR. This is simply a reminder of what we are going to look
+  for before merging your code.
+-->
+- [ ] I've read and followed the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
+- [ ] `step.yml` and `README.md` is updated with the changes (if needed)
+- Requires a *MAJOR/MINOR/PATCH* [version update](https://semver.org/) <!-- Leave this untouched if you don't know, we'll help -->
```

Comment:
![image](https://user-images.githubusercontent.com/13942888/132679991-ba851177-0ea5-4a76-bb6e-8fe8cb6ed126.png)
I'd remove the `- ` because it looks weird that the checkboxes don't have the dot but the thrid line does

Reply:
How about now?

Reply:
Much better